indico/web/client/js/custom_elements/_base.js

```diff
@@ -161,19 +161,26 @@ export default class CustomElementBase extends HTMLElement {
     // and disconnect, and the setup() method is expected to set up
     // listeners that will perform such operations.
 
+    this.unmountController = new AbortController();
     this.setup?.();
     this.setup = null;
     this.dispatchEvent(new Event('x-connect'));
+    this.onconnect?.();
```

Comment:
Is this custom? If yes, let's rename it to `onConnect`, the current capitalization is weird...

Reply:
It's custom, and it's following the convention of the DOM API.

Reply:
ok, fine then